### PR TITLE
Don't include type parameters in output information

### LINKED ISSUE #22: Generic fns parsing
```rust
impl Struct {
    /// This is func
    pub fn func<T: Foo>() { }
}
```
Expected output:
```
=! file.rs
=| impl Struct§fn func
    /// This is func
```
Actual:
```
=! file.rs
=| impl Struct§fn func<T
    /// This is func
```
Affects regenerator too.

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @gkoz 

Fixes #22

### CODE REVIEW

File: src/strip.rs [hunk 1]
```diff
@@ -50,6 +50,21 @@ fn move_until(words: &[&str], it: &mut usize, limit: &str, line: &mut usize) {
...
+            break;
+        }
+        it += 1;
+    }
+    if it < word.len() {
+        &word[0..it]
+    } else {
+        word
+    }
+}
```
Comment (gkoz) [inline src/strip.rs]: This looks very similar to `word.find(limit_chars).map(|pos| &word[..pos]).unwrap_or(word)` where `limit_chars = ["\t", "\n", etc]`.

Comment (GuillaumeGomez) [inline src/strip.rs]: Way better, thanks!

**Comment by GuillaumeGomez** [comment]
It seems that regeneration is broken once again. I'll fix it tomorrow.

File: src/strip.rs [hunk 2]
```diff
@@ -184,7 +188,12 @@ pub fn build_event_list(path: &Path) -> io::Result<ParseResult> {
                         event_list.push(EventInfo::new(line, EventType::Type(TypeStruct::new(Type::from(ty), &name))));
                     }
                     "struct" | "fn" | "enum" | "const" | "static" | "type" | "trait" | "macro_rules!" => {
-                        event_list.push(EventInfo::new(line, EventType::Type(TypeStruct::new(Type::from(words[it]), words[it + 1]))));
+                        event_list.push(EventInfo::new(line, EventType::Type(
+                                                                TypeStruct::new(
+                                                                    Type::from(words[it]),
+                                                                               get_before(words[it + 1],
+                                                                                          &vec!('\t', '\n', '\r', '<', '{', ':', ';'))
```
Comment (gkoz) [inline src/strip.rs]: Vector is not unnecessary.

Comment (GuillaumeGomez) [inline src/strip.rs]: But isn't really necessary either.

Comment (gkoz) [inline src/strip.rs]: Yeah, didn't intend that double negative haha

Comment (GuillaumeGomez) [inline src/strip.rs]: :p

**Pull request merged**